enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -396,6 +397,8 @@ def create_transaction(
             subsidy_access_policy_uuid=subsidy_access_policy_uuid,
             **transaction_metadata,
         )
+        event_bus.send_transaction_created_event(ledger_transaction)
```

Comment:
Would it be useful to add some logging around these event sends for troubleshooting later?

Reply:
There's lots of out-of-the-box logging from openedx-events, it includes the message key, which we have configured to be the transaction uuid in this case.